osbs/api.py

```diff
@@ -200,10 +200,18 @@ def get_build_logs(self, build_id, follow=False, namespace=DEFAULT_NAMESPACE):
             if ex.status_code != 404:
                 raise
         else:
-            if build["status"] in ["Complete", "Failed"]:
-                return build["metadata"]["labels"]["logs"]
-            else:
-                return self.os.logs(build_id, follow=False, namespace=namespace)
+            build_response = BuildResponse(build)
+            # Should this be 'build_response.is_finished()'?
```

Comment:
Ideally, yes. I would be in favor of doing stuff like `build_response.json["status"] in ["Complete", "Failed"]` only on one place. Let's merge and improve in future.